PolyEngine/Core/Src/Logger.hpp

```diff
@@ -23,8 +25,10 @@ namespace Poly
 	inline void sprint(std::ostream& stream, eLogLevel level,
 		const std::string& fmt) {
 		(void)level;  // suppress unused warning
-		stream << fmt << std::endl;
+		stream << fmt.c_str();
+		stream << String("\n"); 
```

Comment:
This line is unnecessary. Remove it.

Reply:
I leave it here, because otherwise we get everything in one line, just checked it

Reply:
I replaced it with std::endl